core-bundle/src/Resources/contao/modules/ModuleTwoFactor.php

```diff
@@ -46,7 +46,7 @@ protected function compile()
 		$user = BackendUser::getInstance();
 
 		// Inform the user if 2FA is enforced
-		if (!$user->useTwoFactor && empty($_GET['act']) && $container->getParameter('contao.security.two_factor.enforce_backend'))
+		if (!$user->useTwoFactor && !Input::get('act') && $container->getParameter('contao.security.two_factor.enforce_backend'))
```

Comment:
```suggestion
		if (!$user->useTwoFactor && null !== Input::get('act') && $container->getParameter('contao.security.two_factor.enforce_backend'))
```

Just a nitpick because this is how you have done it in the other files.

Reply:
I replaced `isset()`s with null checks and `empty()`s with boolen checks to keep the behavior the same.

Reply:
I see. 👍